debian/qubes-vm-pipewire.postinst

```diff
@@ -0,0 +1 @@
+systemctl --global --no-reload preset wireplumber.service pipewire.service pipewire.socket || :
```

Comment:
For any custom maintainer script, please include `#DEBHELPER#` placeholder auto automatically generated parts (see other packages). In fact, you may be able to automatically generate this one too, using [dh_installsystemduser](https://www.man7.org/linux/man-pages/man1/dh_installsystemduser.1.html) - it's enabled by default, but you may need to tell it which units to handle, as you install just drop-ins here, not the actual units.